core/include/thread_flags.h

```diff
@@ -95,13 +95,21 @@ extern "C" {
  *      }
  */
 #define THREAD_FLAG_MSG_WAITING     (1u << 15)
+
 /**
  * @brief Set by @ref xtimer_set_timeout_flag() when the timer expires
  *
  * @see xtimer_set_timeout_flag
  */
 #define THREAD_FLAG_TIMEOUT         (1u << 14)
 
+/**
+ * @brief Thread flag use to notify available events in an event queue
+ *
+ * This flag is used by the `event` module.
+ */
+#define THREAD_FLAG_EVENT           (1u << 13)
```

Comment:
Changing the value from `0x1` to `1 << 13` might break applications that relied on the fact that the latter was unused before.

Reply:
I considered that, but figured that it boiled down to too cases; applications that rely on the flag `1 << 13` being free, and applications that rely on event flags being equal to `0x1`.

In the first case, application should rely in the macro `THREAD_FLAG_PREDEFINED_MASK` to tell them if a flag is free and should be prepared to deal with once free flags becoming no longer free in later versions of RIOT.

In the second case, applications should use `THREAD_FLAG_EVENT` rather than the value of `0x1` directly, and should not be affected.

One artifact of changing the value is that [`thread_flags_wait_one()`](https://doc.riot-os.org/group__core__thread__flags.html#gaf96b4d49f0d34f1ff5c5d07b787d5b9b) does state that the flags are serviced in a specific order according to their value. However again the thread flag's value was re-definable. This means that no part or RIOT itself should have ever relied on this ordering. I suppose you could make an argument that an application could have relied on this, but seems unlikely.

I am fine with it being set to either value. What's you preference?